lib/cargo.js

```diff
@@ -13,10 +22,12 @@ function provideBuilder() {
     },
 
     settings: function (path) {
+      var cargoPath = atom.config.get('build-cargo.cargoPath');
```

Comment:
I think you should default to `'cargo'` here as well, in case the setting variable doesn't exist (e.g. someone overwrites their config file etc).